src/main/kotlin/com/chattriggers/ctjs/minecraft/listeners/ClientListener.kt

```diff
@@ -67,7 +69,18 @@ object ClientListener {
 
     @SubscribeEvent
     fun onTick(event: TickEvent.ClientTickEvent) {
-        if (World.getWorld() == null || event.phase == TickEvent.Phase.END)
+        if (event.phase == TickEvent.Phase.END)
+            return
+
+        tasks.removeAll {
+            if (it.delay-- <= 0) {
+                it.callback()
+                return@removeAll true
+            }
+            false
```

Comment:
Small code cleanliness nitpick
```suggestion
            if (it.delay-- <= 0) {
                it.callback()
                true
            } else false
```